qrenderdoc/Code/CaptureContext.cpp

```diff
@@ -1258,10 +1270,16 @@ WindowingData CaptureContext::CreateWindowingData(uintptr_t widget)
 
 #elif defined(RENDERDOC_PLATFORM_LINUX)
 
+#if defined(RENDERDOC_WINDOWING_WAYLAND)
+  if(m_CurWinSystem == WindowingSystem::Wayland)
+    return CreateWaylandWindowingData(m_WaylandDisplay, (struct wl_surface *)widget);
```

Comment:
The `struct wl_surface *` instead of `wl_surface *` is a C-ism which isn't needed for C++, so I'd rather the `struct` was omitted - the same thing occurs in a few places where these structs are referenced.

Reply:
Sounds good! I'll reference these structs without the explicit keyword.